bullet_train-outgoing_webhooks/test/integration/api/a1/webhooks/outgoing/endpoints_controller_test.rb

```diff
@@ -3,6 +3,9 @@
 class Api::V1::Webhooks::Outgoing::EndpointsControllerTest < ActionDispatch::IntegrationTest
   setup do
     @controller = Api::V1::Webhooks::Outgoing::EndpointsController
+
+    @team = Team.first
+    @endpoint = @team.webhooks_outgoing_endpoints.first
```

Comment:
Grabbing a couple of objects for reference later in the tests. Doing this since we can't (and shouldn't try to) rely on hard coded ids.